### PR TITLE
Issues/229

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: docs/swagger.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ paths:
         - "Host Credential"
```
Comment (chambridge) [inline docs/swagger.yml]: Update the tag to just "Credential"

File: docs/swagger.yml [hunk 2]
```diff
@@ -47,7 +47,7 @@ paths:
...
       summary: "Add a new host credential"
```
Comment (chambridge) [inline docs/swagger.yml]: need to replace every "host credential" -> "credential"

File: quipucords/api/credential/tests_credential.py [hunk 3]
```diff
@@ -53,27 +53,27 @@ def create_expect_201(self, data):
...
         data = {'name': 'cred1',
                 'username': 'user1',
                 'password': 'pass1'}
         self.create_expect_201(data)
-        self.assertEqual(HostCredential.objects.count(), 1)
-        self.assertEqual(HostCredential.objects.get().name, 'cred1')
+        self.assertEqual(Credential.objects.count(), 1)
+        self.assertEqual(Credential.objects.get().name, 'cred1')
 
     def test_hostcred_create_double(self):
```
Comment (chambridge) [inline quipucords/api/credential/tests_credential.py]: update function names "test_hostcred_create" -> "test_cred_create"

File: docs/swagger.yml (hunk 1 above)
Comment (kholdaway) [reply docs/swagger.yml]: Fixed

File: docs/swagger.yml (hunk 2 above)
Comment (kholdaway) [reply docs/swagger.yml]: Fixed

**Pull request merged**